### PR TITLE
Add vsmanproj for edge test templates

### PR DESCRIPTION
**Pull request by Cosifne**
Add the vsmanproj file for edge test templates

@singhsarab  Could you take a look at this short PR?

### CODE REVIEW

File: src/Setup/Microsoft.VisualStudio.Templates.CS.EdgeDriverTestCore.vsmanproj [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (singhsarab) [inline src/Setup/Microsoft.VisualStudio.Templates.CS.EdgeDriverTestCore.vsmanproj]: What is the difference between the EdgeDriverTestCore and EdgeDriverTest?

Comment (Cosifne) [reply src/Setup/Microsoft.VisualStudio.Templates.CS.EdgeDriverTestCore.vsmanproj]: > 
> 
> What is the difference between the EdgeDriverTestCore and EdgeDriverTest?

One is for .Net Framework, one for .Net Core

Comment (Cosifne) [reply src/Setup/Microsoft.VisualStudio.Templates.CS.EdgeDriverTestCore.vsmanproj]: > 
> 
> What is the difference between the EdgeDriverTestCore and EdgeDriverTest?

I have checked with Manish, for those two templates we just want to ship it with VS, currently not for CLI

**Comment by singhsarab** [comment]
What testing have we done around these templates?
Can you please try to run the configured definition on your branch and then check if everything is working fine ?

**Comment by Cosifne** [comment]
> 
> 
> What testing have we done around these templates?
> Can you please try to run the configured definition on your branch and then check if everything is working fine ?

I have test it on my machine via devenv. It should test by vendor. I can create an insertion PR at VS side and do it.

**Pull request merged**